package.json

```diff
@@ -99,18 +101,19 @@
     "enzyme": "^3.8.0",
     "enzyme-adapter-react-16": "^1.7.1",
     "enzyme-to-json": "^3.3.5",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb": "^17.1.0",
-    "eslint-import-resolver-webpack": "^0.8.3",
-    "eslint-loader": "^2.1.1",
```

Comment:
Isn't this necessary to load eslint with webpack? I started getting an error for it (`webpack.config.js` points to `eslint-loader`)

Reply:
Just confirming that, I'll open a PR with some updates, if you want I can do that along with it.

Reply:
My bad... I just removed all the previous `*eslint*` dependencies :grimacing: